integration-tests/src/test/java/com/redhat/parodos/integration/notification/NotificationRecord.java

```diff
@@ -26,42 +26,50 @@ public class NotificationRecord {
 
 	private static final String BASE_PATH = "http://localhost:8081";
 
+	private static final String user = "test";
+
+	private static final String password = "test";
+
 	private NotificationRecordApi recordApiInstance;
 
 	private NotificationMessageApi messageApiInstance;
 
+	private int listNotificationsExpectedCount = 0;
```

Comment:
No, because the actual values in the notification-service itself aren't getting reset between each test
This parameter should have the real count from the server